invoices/sql_migration_test.go

```diff
@@ -371,12 +371,18 @@ func TestMigrateSingleInvoice(t *testing.T) {
 	}
 
 	for _, test := range tests {
+		test := test
```

Comment:
do we still need this although we have the `loopvar` now mandatory ?

Reply:
Yeah not really. This is also changed now after rewriting to use rapid.